chicagoland.pandemicresponsecommons.org/manifest.json

```diff
@@ -20,7 +20,7 @@
     "spark": "quay.io/cdis/gen3-spark:2020.11",
     "hatchery": "quay.io/cdis/hatchery:2020.11",
     "wts": "quay.io/cdis/workspace-token-service:2020.11",
-    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:2020.11",
+    "ssjdispatcher": "quay.io/cdis/ssjdispatcher:1.0.4",
```

Comment:
All new versions of ssjdispatcher have a hard dependency on the `metadata-service` (mds) service. I think the CI tests will continue to fail until the `metadata` version is added to the manifest.
cc: @johnfrancismccann ,

Reply:
Sure I can add it back again, I added it to begin with and removed it

Reply:
I think we need to config all jenkins environment having MDS credentials in ssjdispatcher job block. You don't need to put the MDS service to the manifest

Reply:
@giangbui the [Jenkins environments should all be configured with MDS creds](https://github.com/uc-cdis/cloud-automation/pull/1428#issuecomment-708001359).

Reply:
Wait @johnfrancismccann @giangbui , so the data upload tests (+ssjdispatcher) worked without the `metadata` service after all? 🤔

Reply:
@themarcelor [Jenkins run 4 (which locked jenkins-genomel)](https://jenkins.planx-pla.net/blue/organizations/jenkins/CDIS_GitHub_Org%2Fcdis-manifest/detail/PR-2358/4/tests) did not pass the data upload tests, but [Jenkins run 5 (which locked jenkins-dcp)](https://jenkins.planx-pla.net/blue/organizations/jenkins/CDIS_GitHub_Org%2Fcdis-manifest/detail/PR-2358/5/tests/) did. From looking at the indexing job logs, it looks like run 4 is dispatching a job that attempts to communicate with MDS (i.e. probably `indexs3client:master`), but run 5 is dispatching a job that does not attempt to communicate with MDS (i.e. probably `indexs3client:2020.11`). I don't know exactly what's going on since both runs 4 and 5 use the same `cdis-manifest` commit and in both runs, the ModifyManifest stage  shows that `indexs3client` is being pinned to `2020.11`. It seems that there is something about the `jenkins-genomel` environment causing the `JOB_IMAGES` env variable in the `ssjdispatcher` pod to specify `indexs3client:master` instead of `indexs3client:2020.11`.